README.md

```diff
@@ -45,23 +51,25 @@ may also be helpful.
 
 ## Quick usage
 
+<!---
```

Comment:
Yep you are right 
```
<!---
## Quick usage
```
I understand your point but I think it's more clean to comment it.
The goal is to replace all current README and we already have some of content for part of library and not for others so even if the section is required to align all readme and reduce the work for the first step which will not needed we can comment it. The second step will to ensure all required section is completed. We can create an issue with a list to ensure we have everythings done.